platform/android/java/THIRDPARTY.md

```diff
@@ -3,41 +3,16 @@
 This file list third-party libraries used in the Android source folder,
 with their provenance and, when relevant, modifications made to those files.
 
-## Google's vending library
+## Google's licensing library
 
-- Upstream: https://github.com/google/play-licensing/tree/master/lvl_library/src/main/java/com/google/android/vending
+- Upstream: https://github.com/google/play-licensing/tree/master/lvl_library/
 - Version: git (eb57657, 2018) with modifications
 - License: Apache 2.0
 
-Overwrite all files under `com/google/android/vending`.
+Overwrite all files under:
 
-Modify those files to avoid compile error and lint warning:
+- `aidl/com/android/vending/licensing`
```

Comment:
Regarding those, there was
```
// Android library projects do not yet support AIDL, so this has been
// precompiled into the src directory.
```
in their `.java` version included in the `src/` folder previously. But it seems to work fine now?

Reply:
it works fine, because we are not compiling android **library** project.